test_runner/fixtures/neon_fixtures.py

```diff
@@ -2637,3 +2637,15 @@ def wait_for_last_flush_lsn(env: NeonEnv, pg: Postgres, tenant: uuid.UUID, timel
     """Wait for pageserver to catch up the latest flush LSN"""
     last_flush_lsn = lsn_from_hex(pg.safe_psql("SELECT pg_current_wal_flush_lsn()")[0][0])
     wait_for_last_record_lsn(env.pageserver.http_client(), tenant, timeline, last_flush_lsn)
+
+
+def fork_at_current_lsn(
+    env: NeonEnv,
+    pg: Postgres,
+    new_branch_name: str = DEFAULT_BRANCH_NAME,
+    ancestor_branch_name: Optional[str] = None,
```

Comment:
I think `ancestor_branch_name` be mandatory. This doesn't really make sense without an ancestor branch.

```suggestion
    ancestor_branch_name: str,
```